_schema.graphql

```diff
@@ -237,6 +239,11 @@ type AnalyticsPartnerSalesTimeSeriesStats {
   ): String
 }
 
+type AnalyticsPartnerShow {
+  entityId: String!
+  partnerId: String!
+}
```

Comment:
Just out of curiosity—all these types seem simple join models, is there a reason they can’t all be the same type?